docs/src/content/docs/guides/sidebar.mdx

```diff
@@ -235,13 +235,14 @@ The `sidebar` frontmatter configuration is only used for autogenerated links and
 
 ## Badges
 
-Links can also include a `badge` property to display a badge next to the link label.
+Links and groups can also include a `badge` property to display a badge next to their label.
```

Comment:
Thank, you for the feedback @HiDeoo. Yes, I completely forgot about autogenerated groups. For some reason, I thought they were in the same section as groups but it makes sense for them to have their own section since they have a lot more configurable options.